doc/code/qml_fourier.rst

```diff
@@ -4,6 +4,302 @@ qml.fourier
 This module contains tools for computing Fourier series representations of
 quantum circuits.
 
-.. automodapi:: pennylane.fourier
-    :include-all-objects:
-    :no-inheritance-diagram:
+.. currentmodule:: pennylane.fourier
+
+Overview
```

Comment:
I love the new look of this page! A couple of questions though:

- How come this content was not moved into `fourier/__init__.py` as per https://github.com/PennyLaneAI/pennylane/pull/1160#discussion_r635767019? Was there a technical issue, or did it not feel right in the end?

- It would be nice if it were possible to use `automodapi` here to avoid having to manually keep the autosummary tables up to date, did automodapi stop working in this case?

Reply:
I'm the culprit...I couldn't find a better way.

If the docstring is in the module's `_init_` function and we use `automodapi` then the functions get imported a second time after the doctring, no?

Reply:
Oh wait, you mean just use automodapi here? I'll try!

Reply:
> Oh wait, you mean just use automodapi here, instead of the two sections?

yep!

Reply:
Done!